Common/Securities/Forex/Forex.cs

```diff
@@ -47,7 +47,8 @@ public Forex(SecurityExchangeHours exchangeHours, Cash quoteCurrency, Subscripti
                 new ImmediateSettlementModel(),
                 Securities.VolatilityModel.Null,
                 new SecurityMarginModel(50m),
-                new ForexDataFilter()
+                new ForexDataFilter(),
+                new AdjustedPriceVariationModel()
```

Comment:
If we use SecurityPriceVariationModel() as default, we will change results from previous backtests, since it will start rounding limit/stop orders. Is that what we want?